### PR TITLE
Releasev0.8

### PR DESCRIPTION
**Pull request by HajimeKawahara**
PR for the release v0.8.
See #145.
For discussions of this release, use #147.
New doc is [here](http://secondearths.sakura.ne.jp/exojax/develop/).

### CODE REVIEW

File: src/exojax/spec/moldb.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class MdbExomol(object):
     MdbExomol is a class for ExoMol.
 
     Attributes:
+        nurange: nu range [min,max] (cm-1)
```
Comment (HajimeKawahara) [inline src/exojax/spec/moldb.py:22]: in the commnet
n_Tref -> n_Texp
n_Tref_def -> n_Texp_def

**Comment by HajimeKawahara** [comment]
Performed a test for MODIT HMC w/ and w/o a new folding using [this tutorial](http://secondearths.sakura.ne.jp/exojax/develop/tutorials/MODITrv.html#).
- w/ new folding: 39min
- w/o new folding: 37min
So, the new folding does not a lot affect the computation time even for HMC .

**Comment by ykawashima** [comment]
Thank you very much! I have confirmed that the code works properly for the calculations of cross sections and spectra for several species, and both basic and MODIT cases.

**Review by ykawashima — APPROVED** [review]

**Comment by HajimeKawahara** [comment]
Thank you so much! @ykawashima

**Pull request merged**